components/removed_from_channel_modal/removed_from_channel_modal.jsx

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+import {Modal} from 'react-bootstrap';
+
+import {ModalIdentifiers} from 'utils/constants.jsx';
+
+export default class RemovedFromChannelModal extends React.PureComponent {
+    static propTypes = {
+
+        /**
+         * Object with info about townsquare
+         */
+        currentUser: PropTypes.object.isRequired,
+
+        /**
+         * String with name of currently selected channel
+         */
+        channelName: PropTypes.string,
+
+        /**
+         * String with name of admin who removed currentUser from channel
+         */
+        remover: PropTypes.string,
+
+        /*
+         * Object with redux action creators
+         */
+        actions: PropTypes.shape({
+
+            /*
+             * Action creator to close modal
+             */
+            closeModal: PropTypes.func.isRequired,
+        }).isRequired,
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            show: true,
+        };
+    }
+
+    handleClose = () => {
+        this.setState({show: false});
+    }
+
+    render() {
+        const {currentUser, actions: {closeModal}} = this.props;
+
+        var channelName = (
+            <FormattedMessage
+                id='removed_channel.channelName'
+                defaultMessage='the channel'
+            />
+        );
+
+        if (this.props.channelName) {
+            channelName = this.props.channelName;
+        }
+
+        var remover = (
+            <FormattedMessage
+                id='removed_channel.someone'
+                defaultMessage='Someone'
+            />
+        );
+
+        if (this.props.remover) {
+            remover = this.props.remover;
+        }
+
+        if (currentUser != null) {
+            return (
+                <Modal
+                    show={this.state.show}
+                    onHide={this.handleClose}
+                    onExited={() => closeModal(ModalIdentifiers.REMOVED_FROM_CHANNEL)}
```

Comment:
Can you make this a method of the component so that we can avoid using arrow functions in render? It's typically considered a good practice in React since this actually passes a new function to Modal and causes it to re-render whenever this renders